src/types/UiaTextRangeBase.hpp

```diff
@@ -140,19 +140,22 @@ namespace Microsoft::Console::Types
     public:
         // degenerate range
         HRESULT RuntimeClassInitialize(_In_ IUiaData* pData,
-                                       _In_ IRawElementProviderSimple* const pProvider) noexcept;
+                                       _In_ IRawElementProviderSimple* const pProvider,
+                                       _In_ std::wstring wordDelimiters = L" ");

```

Comment:
I think we have a def of UNICODE_SPACE somewhere and maybe also a `constexpr` string view version of it. Can you use those instead?

I've had a bad experience with L" " before where it can be interpreted as a different space character if the locale of cl.exe/link.exe isn't US-English.